flybywiresim/a32nx
# fix(ecam): All cycle functionality

Fixes #3123

## Summary of Changes

Removed settInterval in favor of using deltatime to fix the page jumping issue. Also switched the statements that control the page and the "active" light of the corresponding button, to keep them better in sync.

## Screenshots (if necessary)



![ezgif-7-d66965f0e0c7](https://user-images.githubusercontent.com/19493808/111085935-fb328200-8519-11eb-835c-582192c2305e.gif)





## Additional context



Discord username (if different from GitHub): saschl#9432

## Testing instructions

1. Click the ecam ALL button once. The page should switch to the next.
2. Keep the ecam all button pressed: Pages should cycle in order and there should be no jumping back and forth
3. Select an ecam page and after that press and hold the ALL button. Pages should cycle normally.




## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
